dashboard/test/models/script_test.rb

```diff
@@ -2270,6 +2272,38 @@ class SummarizeVisibleAfterScriptTests < ActiveSupport::TestCase
     assert_equal 2, script.lesson_groups[1].position
   end
 
+  test 'script levels have the correct chapter and position value' do
+    l1 = create :level
+    l2 = create :level
+    l3 = create :level
+    l4 = create :level
+    new_dsl = <<-SCRIPT
+      lesson_group 'content', display_name: 'Content'
+      lesson 'Lesson1'
+      level '#{l1.name}'
+      level '#{l2.name}'
+
+      lesson_group 'content2', display_name: 'Content'
+      lesson 'Lesson2'
```

Comment:
Could you add more than one lesson in one of the lesson groups to test that case as well?